### PR TITLE
Remove tech preview statement for no worker deployments

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/24777

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio Do you know who might be able to QE this change?

File: modules/nodes-nodes-working-master-schedulable.adoc [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
-default.
+nodes. By default, master nodes are not schedulable. 
 
-[IMPORTANT]
+You can set the masters to be schedulable, but must retain the worker nodes.
+
+[NOTE]
 ====
-In version {product-version}, the ability to create a cluster that does not have worker nodes is available to only clusters that are deployed on bare metal as a technology preview. For all other cluster types, you can set the masters to be schedulable but must retain worker nodes.
+You can deploy {product-title} with no workder nodes on a bare metal cluster.
```
Comment (vikram-redhat) [inline modules/nodes-nodes-working-master-schedulable.adoc:16]: @mburke5678 typo

**Comment by mburke5678** [comment]
@sunilco Can you help with this small change?

**Comment by sunilcio** [comment]
@mburke5678 LGTM, thanks!

File: modules/nodes-nodes-working-master-schedulable.adoc [hunk 2]
```diff
@@ -7,13 +7,14 @@
 
 As of {product-title} 4.2, you can configure master nodes to be
```
Comment (bobfuru) [inline modules/nodes-nodes-working-master-schedulable.adoc:8]: Not in scope for this PR but wondering if you can remove "As of 4.2" now that we support only back to 4.3?

**Review by bobfuru — COMMENTED** [review]
One tangential question but otherwise LGTM.

File: modules/nodes-nodes-working-master-schedulable.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/nodes-nodes-working-master-schedulable.adoc:8]: I thought about removing that. Wanted to see if a reviewer had the same idea. Thanks!!

**Pull request merged**